### PR TITLE
[metasrv] test: write and read works after restarting the cluster

### PR DESCRIPTION
**Pull request by drmingdrmer**
I hereby agree to the terms of the CLA available at: https://databend.rs/policies/cla/

## Changelog

- Improvement
- Build/Testing/CI

## Related Issues

### CODE REVIEW

File: metasrv/tests/it/flight/metasrv_flight_meta_api_restart_cluster.rs [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+use crate::tests::service::start_metasrv_cluster;
+use crate::tests::service::MetaSrvTestContext;
+use crate::tests::start_metasrv_with_context;
+
+#[tokio::test(flavor = "multi_thread", worker_threads = 1)]
+async fn test_meta_api_restart_cluster_write_read() -> anyhow::Result<()> {
+    let (_log_guards, ut_span) = init_meta_ut!();
+    let _ent = ut_span.enter();
+
+    fn mk_key(tc: &MetaSrvTestContext, k: impl std::fmt::Display) -> String {
```
Comment (ariesdevil) [inline metasrv/tests/it/flight/metasrv_flight_meta_api_restart_cluster.rs:38]: make is short enough

Comment (drmingdrmer) [reply metasrv/tests/it/flight/metasrv_flight_meta_api_restart_cluster.rs:38]: You want it longer? All right

File: metasrv/tests/it/flight/metasrv_flight_meta_api_restart_cluster.rs [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+use crate::tests::service::start_metasrv_cluster;
+use crate::tests::service::MetaSrvTestContext;
+use crate::tests::start_metasrv_with_context;
+
+#[tokio::test(flavor = "multi_thread", worker_threads = 1)]
+async fn test_meta_api_restart_cluster_write_read() -> anyhow::Result<()> {
+    let (_log_guards, ut_span) = init_meta_ut!();
+    let _ent = ut_span.enter();
+
+    fn make_ket(tc: &MetaSrvTestContext, k: impl std::fmt::Display) -> String {
```
Comment (ariesdevil) [inline metasrv/tests/it/flight/metasrv_flight_meta_api_restart_cluster.rs:38]: make_ket -> make_key

Comment (drmingdrmer) [reply metasrv/tests/it/flight/metasrv_flight_meta_api_restart_cluster.rs:38]: Hm...damn it

**Pull request merged**